### PR TITLE
Add property for setup files. [#10]

### PR DESCRIPTION
**Pull request by marnen**
Pull request for #10.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "track_id": "gnu-apl",
   "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "test_pattern": "[.]tc$",
+  "setup_files": ["^[.]/test[.]apl$"],
```
Comment (marnen) [inline config.json]: Hmm. Looking at this, it seems like a regex is really ugly if, as I suspect, this is mostly going to be individual filenames. Maybe a shell glob is the right thing...

Comment (kytrinyx) [reply config.json]: Yeah, I think you're probably right.

Want to give it a try with shell globs?

Comment (marnen) [reply config.json]: Sure, let me see what that looks like.

Comment (marnen) [reply config.json]: Just changed to glob syntax. See what you think.

File: config.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "track_id": "gnu-apl",
   "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "test_pattern": "[.]tc$",
+  "setup_files": ["./test.apl"],
```
Comment (marnen) [inline config.json]: This is meant to be shell glob syntax...

File: config.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "track_id": "gnu-apl",
   "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "test_pattern": "[.]tc$",
```
Comment (marnen) [inline config.json]: This would be awesome as a glob instead of a regex, but I understand that that's not what the client expects.

**Comment by kytrinyx** [comment]
I opened a discussion about patterns here: https://github.com/exercism/discussions/issues/192 -- your input would be valuable.

**Comment by kytrinyx** [comment]
This looks good—I would love to get the pattern question sorted before approving this, as things will likely be a bit unstable until we have that worked out.

**Comment by marnen** [comment]
Thanks. I'll try to look at the other discussion.

**Pull request closed without merging**